### ISSUE TITLE
ActiveRecord::StatementInvalid: PG::ConnectionBad: connection is closed when using db:schema:load

### ISSUE THREAD

**Issue by tatyree** [thoughtbot/scenic#129]
El Capitan 
Rails 4.2.5
Postgres 9.4.5 from homebrew

Create a new rails project
Add the scenic gem and bundle 
`bundle exec rails g scenic:view hello_world`
Add anything to the view (in this instance `SELECT 'hello world';`)
`bundle exec rake db:create db:migrate` works as expected
Subsequent calls to 
`bundle exec rake db:drop db:setup` or `bundle exec rake db:drop db:create db:schema:load` fail with:

```ruby
ActiveRecord::StatementInvalid: PG::ConnectionBad: connection is closed: CREATE VIEW "hello_worlds" AS       SELECT 'hello world';
```
However, subsequent `bundle exec rake db:drop db:create db:migrate` calls work.

**Comment by derekprior** [comment]
We do precisely this in our test suite with Rails 4.2.5 and PG 9.4. I'm also using this on my own Rails 4.2.5 projects without issue. I'll give this a shot on Friday to see if I can reproduce here.

**Comment by adamnbowen** [comment]
I can reproduce this issue by following the steps above with Postgres.app at version 9.4.4.0 (I'm also on El Capitan), so it at least shouldn't be Homebrew-specific.

**Comment by AskarZinurov** [comment]
Reproduced on 4.1.14 & PG 9.4

**Comment by derekprior** [comment]
I created a test app to reproduce this: https://github.com/derekprior/scenic_bug

And then I executed the rake tasks with `-t` so I could get a trace:

```
scenic_bug master % bundle exec rake db:reset -t
** Invoke db:reset (first_time)
** Invoke environment (first_time)
** Execute environment
** Invoke db:load_config (first_time)
** Execute db:load_config
** Execute db:reset
** Invoke db:drop (first_time)
** Invoke db:load_config
** Execute db:drop
** Invoke db:setup (first_time)
** Invoke db:schema:load_if_ruby (first_time)
** Invoke db:create (first_time)
** Invoke db:load_config
** Execute db:create
** Invoke environment
** Execute db:schema:load_if_ruby
** Invoke db:schema:load (first_time)
** Invoke environment
** Invoke db:load_config
** Execute db:schema:load
-- enable_extension("plpgsql")
   -> 0.0172s
-- create_view(:hello_worlds, {:sql_definition=>"      SELECT 'hello world';\n"})
WARNING:  column "?column?" has type "unknown"
DETAIL:  Proceeding with relation creation anyway.
   -> 0.0021s
-- initialize_schema_migrations_table()
   -> 0.0240s
-- enable_extension("plpgsql")
   -> 0.0272s
-- create_view(:hello_worlds, {:sql_definition=>"      SELECT 'hello world';\n"})
rake aborted!
ActiveRecord::StatementInvalid: PG::ConnectionBad: connection is closed: CREATE VIEW "hello_worlds" AS       SELECT 'hello world';
```

The odd thing here is that after `initialize_schema_migrations_table()` things start to repeat?

**Comment by derekprior** [comment]
I can't reproduce this with Scenic 1.0.0. Only with Scenic 1.1.0. This narrows the scope a bit.

**Comment by derekprior** [comment]
I pointed my test app at my local check out of scenic and started a `git bisect`. I know that v1.0.0 is good and v1.1.0 is bad. At each step I rand the `db:reset` task to see what the problem was. This identified the commit where the problem was introduced. Turns out it's right here: https://github.com/thoughtbot/scenic/commit/332d6d28016bd67ffcb6db12cdcf425406623ecb#diff-5c0a9160f7877473c255109f580ef941R25

We're saving off an instance of the connection which later gets closed. Whoops. I hope to have a fix released today. If not (or in the meantime) you can use 1.0.0

**Comment by adamnbowen** [comment]
Awesome, thanks for the swift troubleshooting and correction @derekprior :clap:

**Comment by derekprior** [comment]
Closed by #130. I'mm release Scenic 1.1.1 with a fix very shortly...
